toy_data/initial/schema-automator-data/Demographics.yml

```diff
@@ -0,0 +1,115 @@
+name: Toy_Schema
+description: Toy_Schema
+id: https://w3id.org/Toy_Schema
+imports:
+- linkml:types
+prefixes:
+  linkml: https://w3id.org/linkml/
+  Toy_Schema: https://w3id.org/Toy_Schema
+default_prefix: Toy_Schema
+enums:
+  type_enum:
+    permissible_values:
+      demographic:
+        description: demographic
+  project_id_enum:
+    permissible_values:
+      tutorial-synthetic_data_set_1:
+        description: tutorial-synthetic_data_set_1
+  annotated_sex_enum:
+    permissible_values:
+      female:
+        description: female
+      male:
+        description: male
+  race_enum:
+    permissible_values:
+      asian:
+        description: asian
+      white:
+        description: white
+      black or african american:
+        description: black or african american
+      hispanic:
+        description: hispanic
+slots:
+  type:
+    examples:
+    - value: demographic
+    range: type_enum
+  id:
+    examples:
+    - value: 1967affc-04d1-42c1-a21a-39a12ab421cf
+    range: string
+  project_id:
+    examples:
+    - value: tutorial-synthetic_data_set_1
+    range: project_id_enum
+  submitter_id:
+    examples:
+    - value: HG00262_demo
+    range: string
+  age_at_index:
+    examples:
+    - value: '75'
+    range: integer
+  annotated_sex:
+    examples:
+    - value: female
+    range: annotated_sex_enum
+  bmi_baseline:
+    examples:
+    - value: '25.9'
+    range: float
+  height_baseline:
+    examples:
+    - value: '167.5'
+    range: float
+  population:
+    examples:
+    - value: GBR
+    range: string
+  race:
+    examples:
+    - value: white
+    range: race_enum
+  subjects.id:
+    examples:
+    - value: 34ea8f4c-9999-4ee1-8381-672b287861ec
+    range: string
+  subjects.submitter_id:
+    examples:
+    - value: HG00262
+    range: string
+classes:
+  demographics:
+    slots:
```

Comment:
I'm confused why Demographics.yml is a separate file. Demographics also appear to be included in the toy_data-All.yml file and I don't see any clear differences. Was this added intentionally? I see that creating the single data file is mentioned in the README. Do you think that we need both?

Reply:
I don't feel strongly about this so if you think we need it just mention that here and I'll change the review to approved. Thanks! It looks great!

Reply:
Without being able to validate the schema or have some measure of correctness of the schema generation, I thought it would be useful to see what SchemaAutomator creates for one file vs. a list of files in case that makes a different later on.

Reply:
Okay, that sounds fine to me. I think we'll want to consider removing it in the future... On the other hand, hopefully, we'll be moving on to the other toy data set and we can just ignore this.